Source/GUI/MainWindow.cpp

```diff
@@ -384,7 +414,7 @@ void MainWindow::onSplitterMoved(const int pos, const int index)
 void MainWindow::onOpenWatchFile()
 {
   if (m_watcher->warnIfUnsavedChanges())
-    m_watcher->openWatchFile();
+    m_watcher->openWatchFile(QString(""));
```

Comment:
Nit: The use of `""` in superfluous:
```suggestion
    m_watcher->openWatchFile(QString());
```

Reply:
Done